### PR TITLE
Story 25

### PR DESCRIPTION
**Pull request by agallant121**
Complete user story 25

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -29,6 +29,9 @@
       <% else %>
         <%= link_to "Register as New User", "/register" %>
         <%= link_to "Login", "/login" %>
+        <% if cart.total_items > 0 %>
+          <%= "Warning: You must register or log in to finish the checkout process" %>
```
Comment (agallant121) [inline app/views/layouts/application.html.erb:33]: Add conditional to show this statement if there are items in the cart and the user is not registered or logged in

File: spec/features/cart/increment_cart_spec.rb [hunk 2]
```diff
@@ -83,26 +83,12 @@
...
         expect(current_path).to eq('/cart')
       end
-
+      
       expect(page).to have_content("Item has been removed from the cart")
       expect(page).not_to have_css("cart-item-#{@paper.id}")
     end
-  end
 
-  # User Story 24, Decreasing Item Quantity from Cart
```
Comment (agallant121) [inline spec/features/cart/increment_cart_spec.rb:92]: Removed commented out user stories at the bottom of the spec

File: spec/features/cart/increment_cart_spec.rb [hunk 3]
```diff
@@ -87,22 +87,38 @@
...
-  # When I have items in my cart
-  # And I visit my cart
-  # Next to each item in my cart
-  # I see a button or link to increment the count of items I want to purchase
-  # I cannot increment the count beyond the item's inventory size
+    it 'I see information telling me I must register or log in to finish checking out' do
+      expect(page).to have_content("Warning: You must register or log in to finish the checkout process")
+    end
+
+    it "Does not show warning message if logged in" do
```
Comment (agallant121) [inline spec/features/cart/increment_cart_spec.rb:95]: Added a logged in user to make sure the warning message to log in did not appear if they had items in their cart

**Comment by palworth** [comment]
Story 25 ready for master.

**Pull request merged**